main.tex

```diff
@@ -117,21 +117,21 @@ \subsection{Input Registration}
 % - single pedersen multicommitment for amount and serial or two separate group attributes?
 % - if separate, extra generator + randomness for unconditional hiding of serial number even after revealing serial?
 
-The coordinator verifies the proofs, and issues $k$ MACs (message authentication codes) on the requested attributes, along with a proof of knowledge of the secret key as described in \textit{Credential Issuance} protocol of \cite{chase2019signal}.
+The coordinator verifies the proofs, and issues $k$ MACs on the requested attributes, along with a proof of knowledge of the secret key as described in \textit{Credential Issuance} protocol of \cite{chase2019signal}.
 
 \subsection{Output Registration}
 
 Now acting as Bob, to register her output the user randomizes the attributes and generates a proof of knowledge of a valid credential issued by the coordinator.
 
-Additionally, she proves knowledge of representation of the serial number commitments. These serial numbers are revealed for double spending protection, but the knowledge of commitment opening should be done in zero knowledge to avoid revealing the randomness of the original commitment in the input registration phase or the randomization added in output registration time.
+Additionally, she proves knowledge of representation of the serial number commitments. These serial numbers are revealed for double spending protection, but the knowledge of commitment opening must be done in zero knowledge to avoid revealing the randomness of the original commitment in the input registration phase or the randomization added in output registration time.
 
 Finally, she proves that the sum of the randomized amount attributes $C_v$ matches the requested output amount $v_{\mathit{out}}$, analogously to input registration. Note that there is no need for range proofs at this phase.
 
 The user submits these proofs, the randomized attributes, and the serial numbers. The coordinator verifies the proofs, and if it accepts the output will be included in the transaction.
 
 \subsection{Signing phase}
 
-The user fetches the finalized but unsigned transaction as Satoshi, and if she sees her registered outputs she will sign, submitting the signature as Alice(s).
+The user fetches the finalized but unsigned transaction as Satoshi, and if she sees her registered inputs and outputs she will sign, submitting the signature as Alice(s).
```

Comment:
If Alice sees only her outputs, but not the inputs, then she cannot sign the tx. But she still gets the money. So, someone in the transaction is loosing to the benefit of Alice, and that user won't sign the transactoin.

Reply:
At this point he only cares about outputs.

Reply:
Original was correct. It's a research doc, so have to be specific.

Reply:
ACK - I'm reverting

```suggestion
The user fetches the finalized but unsigned transaction as Satoshi, and if she sees her registered outputs she will sign, submitting the signature as Alice(s).
```